doc/user_guide/visualisation.rst

```diff
@@ -955,8 +955,8 @@ This can be extended to 4 (or more) navigation dimensions:
     >>> marker = hs.markers.point(x=x, y=y, color='red')
     >>> s.add_marker(marker, permanent=True)
 
-Many markers can added by using a for loop. Note, adding many markers
-might lead to very slow plotting.
+Many markers can added by using a for loop. Note, adding many (50+) markers
```

Comment:
What about simply removing this from the User Guide: users in a hurry may miss the next paragraph which should be the preferred method in most cases.